src/enhetsportefolje/brukerliste.css

```diff
@@ -142,6 +142,10 @@
     border-radius: 0 0 4px 4px;
     margin-bottom: 0;
 }
+.veilarbportefoljeflatefs .avvik14avedtak-kolonne {
+    hyphens: auto;
+    overflow-wrap: break-word;
+}
```

Comment:
hva gjør denne?

Reply:
Vi brukar desse to propertiene på kolonner der lengden på tekstinnholdet potensielt kan "flyte ut" av kontaineren sin. Antakeligvis er det satt en `max-width` på kolonnene (litt usikker) noko som resulterer i at dette blir tilfelle med lang tekst. Eit alternativ kunne eventuelt vert å gjere denne spesifikke kolonna breiare, men det ville ikkje nødvendigvis løyst problemet om ein forstørrar nettlesarfonten/ aukar side-zoomen. Eit anna alternativ ville vore å ha fullstendig dynamisk bredde på kolonnene. 

`overflow-wrap: break-word` forteller til nettleseren at den får lov til å "brekke" lange ord som "flyter ut" av inline containeren sin. Ein kan [lese meir om `overflow-wrap`-propertien på MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap).

`hypens: auto` forteller nettleseren at dersom den skal brekke et ord i en eller flere deler, så kan den få plassere hyphen (= bindestrek) automatisk. Ein kan [lese meir om `hyphens`-propertien på MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens).